repo/packages/H/hdfs/0/config.json

```diff
@@ -11,7 +11,8 @@
           "type": "string"
         },
         "default": [
-          "https://s3.amazonaws.com/downloads.mesosphere.io/hdfs/artifacts/dcos/0.1.1/hdfs-mesos-0.1.1.tgz"
+          "https://s3.amazonaws.com/downloads.mesosphere.io/hdfs/artifacts/dcos/0.1.1/hdfs-mesos-0.1.1.tgz",
```

Comment:
I'm not sure how the upgrade for this works.  At what point will this hdfs-mesos package be the correct one from [the other PR](https://github.com/mesosphere/hdfs/pull/157)?